js/app.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+console.log('js is connected.');
+
+// Create an algorithm that will randomly generate three unique product images from the images directory and display them side-by-side-by-side in the browser window.
+
+// For each of the three images, increment its property of times it has been shown by one.
+
+// Attach an event listener to the section of the HTML page where the images are going to be displayed.
+
+// Once the users ‘clicks’ a product, generate three new products for the user to pick from.
+// As a user, I would like to track the selections made by viewers so that I can determine which products to begin production on.
+// In the constructor function define a property to hold the number of times a product has been clicked.
+
+// After every selection by the viewer, update the newly added property to reflect if it was clicked.
+
+// ???
+// let pizzaContainer = document.querySelector('section');
+// let resultButton = document.querySelector('section + div');
+// let image1 = document.querySelector('section img:first-child');
+// let image2 = document.querySelector('section img:nth-child(2)');
+
+
+let clicks = 0;
+let maxAttemptsAllowed = 25;
+Pizza.allPizzasArray = [];
+//why is this array in the Pizza constructor obj?
+
+function Pizza(productName, imgFilePath) {
+    this.productName = productName;
+    this.imgFilePath = imgFilePath;
+    this.timesShown = 0;
+    this.click = 0;
+    Pizza.allPizzasArray.push(this);
+}
+
+function getRandomNumber() {
+    return Math.floor(Math.random() * Pizza.allPizzasArray.length);
+    //-1? bc array?
+}
+
+new Pizza('Brick Oven Pizza', 'images/brickOvenPizza.jpg');
+new Pizza('Calzone', 'images/calzonePizza.jpg');
+new Pizza('Chicago Deep Dish', 'images/chicagoPizza.jpg');
+new Pizza('Chicago Pizza and Oven Grinder', 'images/cpoGinderPizza.jpg');
+new Pizza('Detroit Style', 'images/detroitPizza.jpg');
+new Pizza('Papa Vito\'s Thin', 'images/mwDeluxePizzaThinCrust.jpg');
+new Pizza('New York Thin', 'images/newYorkPizza.jpg');
+new Pizza('Shot Gun Dans Pizza', 'images/sgDansHtossedMeatLovPizza.jpg');
+
+// console.log('all pizzas array', Pizza.allPizzasArray)
+
+
+// function renderPizzas() {
+//     let pizza1 = Pizza.allPizzasArray[getRandomNumber()];
+//     let pizza2 = Pizza.allPizzasArray[getRandomNumber()];
+//     let pizza3 = Pizza.allPizzasArray[getRandomNumber()];
+
+//     while (pizza1 === pizza2) {
+//         pizza2 = Pizza.allPizzasArray[getRandomNumber()]
+//     };
+//     while (pizza1 === pizza3) {
+//         pizza3 = Pizza.allPizzasArray[getRandomNumber()]
+//     };
+//     while (pizza2 === pizza3) {
+//         pizza3 = Pizza.allPizzasArray[getRandomNumber()]
+//     };
+
+
+// }
+
+
+
+// console.log(Pizza.allPizzasArray[0])
+
+function renderPizzas() {
+
```

Comment:
function renderProducts (or you can say Items, as long as the names are the same everywhere else, just something that makes sense to what you're doing).

Then filter through those images randomly:

Ex.
  let product1 = getRandomNum();
  let product2 = getRandomNum();
  let product3 = getRandomNum();